index.html

```diff
@@ -75,6 +75,7 @@
     <script type="text/javascript" src="./js/classes/entities/midground/border.js"></script>
     <script type="text/javascript" src="./js/classes/entities/midground/chad.js"></script>
     <script type="text/javascript" src="./js/classes/entities/midground/portal.js"></script>
+    <script type="text/javascript" src="./js/classes/entities/midground/transport_portal.js"></script>
```

Comment:
We wanted to keep this in alphabetical order, right?

Reply:
Oh yeah, I forgot about that. I'll fix the ordering